### PR TITLE
change names

### PR DESCRIPTION
**Pull request by Franavarro**

### CODE REVIEW

**Review by jeremias-adm — CHANGES_REQUESTED** [review]
I disagree

File: README.md [hunk 1]
```diff
@@ -8,13 +8,14 @@ Visit the site here: [Math Utils](https://cristianemoyano.github.io/math_utils/)
 
 This is a web application created for academic purposes.
 
-Subject: Programming II
-University: Champagnat University
+Date: 2020/10/6.
```
Comment (cristianemoyano) [inline README.md:11]: I don't agree about this change. I suggest the following:
`Date: October 2020`

**Review by cristianemoyano — COMMENTED** [review]
I left just a nit comment.

**Review by cristianemoyano — APPROVED** [review]

**Review by jeremias-adm — APPROVED** [review]
LGTM

**Pull request merged**